src/components/package/package-view/package-view.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
Let's put this branching logic for the different `View` and `initialValues` in `src/components/package/edit.js` instead of creating this new `package-view` component. That was the intention of `src/components/package/edit.js` - I'm not sure at what point we diverged from that.

Reply:
Sounds good.